node/node.go

```diff
@@ -60,6 +62,15 @@ var _ facade.NodeHandler = (*Node)(nil)
 //  over the None struct.
 type Option func(*Node) error
 
+// QueryP2PPeerInfo represents a DTO used in exporting p2p peer info after a query
+type QueryP2PPeerInfo struct {
+	Pid           string   `json:"pid"`
+	Addresses     []string `json:"addresses"`
+	Pk            string   `json:"pk"`
+	IsBlacklisted bool     `json:"isblacklisted"`
```

Comment:
move the bool field to the top or the bottom (memory allign reasons)

Reply:
done